query/graphql/src/main/java/org/codice/ddf/admin/graphql/servlet/ExtendedOsgiGraphQLServlet.java

```diff
@@ -240,11 +240,11 @@ private synchronized void refreshSchema() {
     }
 
     public void bindFieldProvider(FieldProvider fieldProvider) {
-        triggerSchemaRefresh(String.format(BINDING_FIELD_PROVIDER, fieldProvider.fieldTypeName()));
+        triggerSchemaRefresh(String.format(BINDING_FIELD_PROVIDER, fieldProvider.getTypeName()));
     }
 
     public void unbindFieldProvider(FieldProvider fieldProvider) {
-        triggerSchemaRefresh(String.format(UNBINDING_FIELD_PROVIDER, fieldProvider == null ? "" : fieldProvider.fieldTypeName()));
+        triggerSchemaRefresh(String.format(UNBINDING_FIELD_PROVIDER, fieldProvider == null ? "" : fieldProvider.getTypeName()));
```

Comment:
As far as I know, yes. I was receiving a null value for the fieldProvider when it was unbinding. Not sure why this happens sometimes and not others but that fieldProvider does need to be removed from the schema regardless.